src/Sylius/Bundle/PayumBundle/Request/GetStatus.php

```diff
@@ -20,6 +20,7 @@ class GetStatus extends BaseGetStatus
 {
     /**
      * @psalm-suppress NonInvariantDocblockPropertyType
+     * @phpstan-ignore-next-line
```

Comment:
What kind of error did you get that you need to ignore it?

Reply:
```
PHPDoc type string of property Sylius\Bundle\PayumBundle\Request\GetStatus::$status is not covariant with PHPDoc type int of overridden property
         Payum\Core\Request\BaseGetStatus::$status.
         💡 You can fix 3rd party PHPDoc types with stub files:
            https://phpstan.org/user-guide/stub-files
```
Payum annotates the property as an int, but it is used as a string here. Obviously, it is not so easy to change here to an int and it is not the purpose of this PR.